kadas/app/milx/kadasmilxexportdialog.cpp

```diff
@@ -0,0 +1,95 @@
+/***************************************************************************
+  kadasmilxexportdialog.cpp
+  --------------------------------------
+  Date                 : September 2024
+  Copyright            : (C) 2024 by Damiano Lombardi
+  Email                : damiano@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+
+#include <qgis/qgsproject.h>
+
+#include "kadasapplication.h"
+#include "kadasmainwindow.h"
+#include "kadas/gui/milx/kadasmilxlayer.h"
+#include "milx/kadasmilxexportdialog.h"
+
+KadasMilxExportDialog::KadasMilxExportDialog( QWidget *parent )
+  : QDialog( parent )
+{
+  ui.setupUi( this );
+
+  setExportMode( ExportMode::Milx );
+
+  ui.comboCartouche->addItem( tr( "Don't add" ) );
+
+  for ( QgsMapLayer *layer : QgsProject::instance()->mapLayers().values() )
```

Comment:
```suggestion
  const QList<QgsMapLayer> layers = QgsProject::instance()->mapLayers().values();
  for ( QgsMapLayer *layer : layers )
```

@3nids you also adjusted that in a last PR?